### PR TITLE
Document same-server-fetch-interval

### PR DESCRIPTION
**Pull request by polyzen**
https://github.com/rss2email/rss2email/pull/60

### CODE REVIEW

File: r2e.1 [hunk 1]
```diff
@@ -185,6 +185,8 @@ Set this to default To email addresses.
 Set an HTTP proxy (e.g. 'http://your.proxy.here:8080/')
 .IP feed-timeout
 Set the timeout (in seconds) for feed server response
+.IP same-server-fetch-interval
+Specify a sleep interval between consecutive fetches from the same server
```
Comment (polyzen) [inline r2e.1:189]:
```suggestion
Specify a sleep interval (in seconds) between consecutive fetches from the same server
```

**Comment by auouymous** [comment]
The comment in config.py should be changed to match man page, as it does not mention the fetches are "consecutive".

**Comment by polyzen** [comment]
They now match. 👍

**Comment by auouymous** [comment]
Thank you.

**Pull request merged**